deployment-local/local-tg

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+if [ "$1" = "startup-jenkins" ]; then
```

Comment:
What does the user expect to see here? When I ran this command, following log was output. As a first-time user, this kept me wondering what I was supposed to do next.

![image](https://user-images.githubusercontent.com/936037/65875001-41d30180-e3a4-11e9-8763-516e8bc30155.png)